index.html

```diff
@@ -33,11 +33,16 @@ <h3>Welcome</h3>
 <p>The Corosync Cluster Engine is a Group Communication System with additional features for implementing high availability within applications. The project provides four C Application Programming Interface features:</p>
 
 <ul>
-<li>A closed process group communication model with virtual synchrony guarantees for creating replicated state machines.</li>
+<li>A closed process group communication model with extended virtual synchrony guarantees
+    (excluding its strongest notion for the time being) for creating replicated state machines.</li>
 <li>A simple availability manager that restarts the application process when it has failed.</li>
 <li>A configuration and statistics in-memory database that provide the ability to set, retrieve, and receive change notifications of information.</li>
 <li>A quorum system that notifies applications when quorum is achieved or lost.</li>
-</ul><p>Our project is used as a High Availability framework by projects such as Apache Qpid and Pacemaker.</p>
+</ul>
+<p>Our project is used as a High Availability foundation in projects such as
+   <a href="https://clusterlabs.org/pacemaker/">Pacemaker</a> (as a prerequisite)
+   or <a href="https://wiki.asterisk.org/wiki/display/AST/Corosync">Asterisk</a>
+   (optionally).</p>
 
```

Comment:
Why to bother with adding prerequisite/optionally? This is not something what should be noted in corosync context. It should be mentioned in project context. Also framework sounds better than foundation.

Reply:
That was meant to enlighten random observers possibly knowing about
Asterisk but not ever hearing of Corosync.  These two examples carry
also another, hidden dichotomy: there's not many publicly known uses
beyond _the_ cluster stack (Qpid previously fulflled this "equal-opportunity"
role).

Reply:
Regarding foundation vs. framework, it's a matter of subjective
perception, but what I expect from "framework" is something that
addresses given area in breadth.  Therefore, from _HA framework_,
I'd expect, e.g., also these:
- something to react to failures of my application
- something in charge of disaster prevention and recovery
  (putting watchdog aside)
- ... (something to assist with cluster-wide execution plan,
  scaling, orchestration, ...)

So while when paired with pacemaker, it offers some sort of this
framework, corosync is not exactly a _HA framework_ alone
(in contrast to _distributed messaging and membership framework_)

I must admit, corosync provides some extra services that, I am
not sure why, are not employed in pacemaker (SAM healthchecking),
since it doesn't install any explicit liveness measures on its
own.

Hopefully my reasoning is clear, it's not about what sounds better.

Reply:
Yes, it's matter of subjective perception, and my subjective perception is that corosync is HA framework.

Reply:
Then, what about `clustering framework`, which addresses your inclination
and, the same time, doesn't promise full-blown HA handling per se.

After all, it may be used just to implement load balancing.

Reply:
FWIW, this is what I'd be less tentative to associate with `framework`:
https://akka.io/
because of the _breadth_ (including various higher level abstractions,
see https://doc.akka.io/docs/akka/current/cluster-usage.html).